src/interfaces/state-interface.ts

```diff
@@ -55,5 +65,12 @@ export interface StateIF {
   hasBlockerTask: boolean
   hasBlockerFiling: boolean
   isCoaPending: boolean
-  coaEffectiveDate: Date
+  coaEffectiveDate: Date,
+  stateFiling: {
+    business: any,
+    header: ApiHeaderIF,
+    dissolution?: any,
+    restoration?: any
+    // any others?
```

Comment:
@severinbeauvais I'm not sure I'm aware of all the stateFiling types there could be

Reply:
Put Back On.

I think that's it, but double check with Mihai or Argus.